data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
> ah wait, this is not actually possible. And we now have libdir twice :(

`libdir` might not be the only value. There are also our `collectives`, witch are noop atm. IMHO we should make it hash configs *or* our parameters.